src/main/java/com/coreos/jetcd/lease/LeaseTimeToLiveResponse.java

```diff
@@ -0,0 +1,79 @@
+package com.coreos.jetcd.lease;
+
+import com.coreos.jetcd.data.ByteSequence;
+import com.coreos.jetcd.data.Header;
+import java.util.List;
+
+public class LeaseTimeToLiveResponse {
+
+  private Header header;
+  private long leaseId;
+  private long ttl;
+  private long grantedTtl;
+  private List<ByteSequence> keys;
+
+  public LeaseTimeToLiveResponse(Header header, long leaseId, long ttl, long grantedTtl,
+      List<ByteSequence> keys) {
+    this.header = header;
+    this.leaseId = leaseId;
+    this.ttl = ttl;
+    this.grantedTtl = grantedTtl;
+    this.keys = keys;
+  }
+
+  public Header getHeader() {
+    return header;
+  }
+
+  public long getLeaseId() {
+    return leaseId;
+  }
+
+  public long getTTl() {
+    return ttl;
+  }
+
+  public long getGrantedTTL() {
+    return grantedTtl;
+  }
+
+  public List<ByteSequence> getKeys() {
+    return keys;
+  }
+
+  @Override
+  public boolean equals(Object o) {
+    if (this == o) {
+      return true;
+    }
+    if (o == null || getClass() != o.getClass()) {
+      return false;
+    }
+
+    LeaseTimeToLiveResponse that = (LeaseTimeToLiveResponse) o;
+
+    if (getLeaseId() != that.getLeaseId()) {
+      return false;
+    }
+    if (ttl != that.ttl) {
+      return false;
+    }
+    if (grantedTtl != that.grantedTtl) {
+      return false;
+    }
+    if (!getHeader().equals(that.getHeader())) {
+      return false;
+    }
+    return getKeys().equals(that.getKeys());
+  }
+
+  @Override
+  public int hashCode() {
```

Comment:
i understand why it is good in some cases to overwrite this. But i feel we do not really need this in our case? less code == less to maintain and worry about.